source/midi_syxg50_control_v2.jsfx

```diff
@@ -1261,6 +1538,7 @@ loop(16,
 	file_var(0,effect_reverb_send[var_offset]);
 	file_var(0,effect_chorus_send[var_offset]);
 	file_var(0,effect_variation_send[var_offset]);
+	file_var(0,voice_dry_level[var_offset]);
```

Comment:
This would break compatibility with data from previous versions, the order that parameters are serialized/deserialized should ideally remain unchanged, new multi-part parameters should be in a separate loop (allowing the previous bulk of already existing parameters to load beforehand).

However, I like that the parameters are now properly organized in a single loop and considering we might end up doing more breaking changes for v3 we could use the opportunity and include this PR there. What do you think?

Reply:
Oh ok, if the order matters then I will add things in a separate loop at the end. I think it's nice to allow people to have an extended 2.x release before we break the Detune/Delay Time stuff in 3.0.